src/components/markdown-renderer/markdown-extension/asciinema/replace-asciinema-link.ts

```diff
@@ -15,16 +13,12 @@ const tailRegex = /(?:[./?#].*)?/
 const gistUrlRegex = new RegExp(`(?:${protocolRegex.source}${domainRegex.source}${idRegex.source}${tailRegex.source})`)
 const linkRegex = new RegExp(`^${gistUrlRegex.source}$`, 'i')
 
-export const asciinemaMarkdownItPlugin: MarkdownIt.PluginSimple = (markdownIt: MarkdownIt) => {
-  markdownItRegex(markdownIt, replaceAsciinemaLink)
-}
-
 export const replaceAsciinemaLink: RegexOptions = {
   name: 'asciinema-link',
   regex: linkRegex,
   replace: (match) => {
     // ESLint wants to collapse this tag, but then the tag won't be valid html anymore.
     // noinspection CheckTagEmptyBody
-    return `<app-asciinema id="${match}"></app-asciinema>`
+    return `<app-asciinema id='${match}'></app-asciinema>`
```

Comment:
shouldn't this `app-asciinema` and the `app-asciinema` in `src/components/markdown-renderer/markdown-extension/asciinema/asciinema-markdown-extension.ts` be the same?